### PR TITLE
Feature/2627 begin writing tests.

### PR DESCRIPTION
**Pull request by owenmorrill**
#2627

### CODE REVIEW

**Comment by owenmorrill** [comment]
Getting this when I try to run test locally:  Mink instance has not been set on Mink context class. Have you enabled the Mink Extension? (RuntimeException) Attempting to debug.

File: modules/custom/express_content_edit_lock/express_content_edit_lock.module [hunk 1]
```diff
@@ -80,11 +81,7 @@ function express_content_edit_lock_admin_form_submit($form, &$form_state) {
...
   // Exclude unnecessary elements.
   form_state_values_clean($form_state);
-  $allowed_users = explode(',', $form_state['values']['lock_allow_users']);
-
-  foreach ($allowed_users as $user) {
-    trim($user);
-  }
+  $allowed_users = explode(', ', $form_state['values']['lock_allow_users']);
 
   $form_state['values']['lock_allow_users'] = $allowed_users;
```
Comment (alexfinnarn) [inline modules/custom/express_content_edit_lock/express_content_edit_lock.module]: This seems weird that a string is turned into an array and inserted back into the form values.

**Comment by owenmorrill** [comment]
Need to fix how we check for allowed users. Changes have to be made otherwise the tests fail because the form submit message never displays.

**Comment by owenmorrill** [comment]
Obviously need a test for allowed_users.

File: modules/custom/express_content_edit_lock/express_content_edit_lock.module [hunk 2]
```diff
@@ -141,12 +133,10 @@ function express_content_edit_lock_secure_permissions($role) {
  * Check to see if user->name is in lock_allow_users variable.
  */
 function _lock_allow_users_check($username) {
-  $output = FALSE;
   $allowed_users = variable_get('lock_allow_users', array());
 
-  if (in_array($username, $allowed_users)) {
-    $output = TRUE;
-  }
+  $output = (in_array($username, $allowed_users)) ? true : false;
```
Comment (alexfinnarn) [inline modules/custom/express_content_edit_lock/express_content_edit_lock.module]: this can be `return in_array($username, $allowed_users);`

**Pull request closed without merging**